### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by kgaillot**
This is a collection of small documentation corrections and clarifications.

### CODE REVIEW

**Comment by kgaillot** [comment]
@waltdisgrace , can you review this when you have time? Check for spelling and syntax errors, confusing wording, etc. You can assume the content itself is correct. If you want to see a final proof, it is available at http://people.redhat.com/~kgaillot/pacemaker/doc/

**Comment by kgaillot** [comment]
@HideoYamauchi , d962d5d is my suggestion for CLBZ#5473, and 5fc6046 is my suggestion for CLBZ#5475

**Comment by HideoYamauchi** [comment]
Hi Ken,

> @HideoYamauchi , [d962d5d](https://github.com/ClusterLabs/pacemaker/commit/d962d5dc3822d84f2294051ef14eb3174f1acc92) is my suggestion for CLBZ#5473, and [5fc6046](https://github.com/ClusterLabs/pacemaker/commit/5fc60468222fdb6d7ec9e5596002c65492a98dfb) is my suggestion for CLBZ#5475

Okay!
I will check the contents.

Many tanks,
Hideo Yamauchi.

**Comment by HideoYamauchi** [comment]
Hi Ken,

Thank you for fixing the documentation.

I talked to a Japanese community member.
Regarding the disable-timeout function, even the fence-agent would like to discuss it a little, so could you postpone the inclusion of the description for a while?

There is no problem with the description of pcmk_host_check.

Best Regards,
Hideo Yamauchi.

**Comment by kgaillot** [comment]
> Hi Ken,
> 
> Thank you for fixing the documentation.
> 
> I talked to a Japanese community member.
> Regarding the disable-timeout function, even the fence-agent would like to discuss it a little, so could you postpone the inclusion of the description for a while?
> 
> There is no problem with the description of pcmk_host_check.
> 
> Best Regards,
> Hideo Yamauchi.

Hi Hideo,

Sure, I will wait until I hear more from you.

**Comment by HideoYamauchi** [comment]
Hi Ken,

> Sure, I will wait until I hear more from you.

Sorry...Thanks!

Hideo Yamauchi.

File: doc/sphinx/Pacemaker_Explained/fencing.rst [hunk 1]
```diff
@@ -205,29 +226,22 @@ for ``pacemaker-fenced``.
...
-   |                      |         | status action but  |                                        |
-   |                      |         | not the list       |                                        |
-   |                      |         | action. If none of |                                        |
-   |                      |         | those conditions   |                                        |
-   |                      |         | apply, the default |                                        |
-   |                      |         | is ``none``.       |                                        |
+   | pcmk_host_check      | string  | Value appropriate  | .. index::                             |
+   |                      |         | to other           |    single: pcmk_host_check             |
+   |                      |         | parameters (see    |                                        |
+   |                      |         | "Default Check     | How to determine which machines are    |
```
Comment (waltdisgrace) [inline doc/sphinx/Pacemaker_Explained/fencing.rst:232]: The description of ```pcmk_host_check``` was not as clear to me as ```pcmk_host_list``` or ```pcmk_host_map```, for example. Here's what I suggest as a possible improvement: "A value that determines which machines are controlled by the device. Allowed values: [...]".

**Comment by waltdisgrace** [comment]
@kgaillot Besides the comment I left, these improvements look good to me.

File: doc/sphinx/Pacemaker_Explained/fencing.rst (hunk 1 above)
Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/fencing.rst:232]: Good point. How about this: "The method Pacemaker should use to determine which nodes can be targeted by this device."

**Comment by HideoYamauchi** [comment]
Hi Ken,

The concerns of our members have also been cleared.
I have no opinion on the documentation.

Many thanks,
Hideo Yamauchi.

**Comment by kgaillot** [comment]
I've rebased on current master and added 9d3d532 to clarify concurrent-fencing based on discussions in CLBZ#5473

**Comment by HideoYamauchi** [comment]
Hi Ken,

> I've rebased on current master and added [9d3d532](https://github.com/ClusterLabs/pacemaker/commit/9d3d532c1705a859dba77638643c0ed3a84267e8) to clarify concurrent-fencing based on discussions in CLBZ#5473

Okay!
Thanks!

**Pull request merged**